pom.xml

```diff
@@ -24,8 +26,6 @@
             <artifactId>jira-rest-java-client-core</artifactId>
             <version>5.2.4</version>
         </dependency>
-        <!-- Update spring-beans version required by jira client to fix security issues
-             CVE-2022-22965, CVE-2016-1000027, CVE-2021-22118, CVE-2021-22096, CVE-2022-22950, CVE-2022-22965 -->
```

Comment:
Is this not necessary any more?

Reply:
Hi  @kaklakariada -- this was fixed in issue #233 by updating spring-beans but forgot to remove comment at that time :)

Reply:
👍